docs/developer/running-saleor/debugging-emails.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Debugging emails
+sidebar_label: Debugging emails
+---
+
+[Saleor platform](https://github.com/mirumee/saleor-platform) use [Mailhog](https://github.com/mailhog/MailHog) for catching all the emails send by the Saleor. 
+
+![Assign product to collection](../screenshots/mailhog.png)
+
+Pros of using Mailhog during development:
+- live preview of rendered emails.
+- no need to send emails to external services.
+- test emails will be caught in the local environment, so there is no possibility of sending test data to actual customers.
```

Comment:
Possibly, we could add that it also prevent hurting the mail reputation due to bounces. e.g. sending emails to `foo@example.com`, which will cause the email to bounce and thus lower the reputation